bazel/dependencies.bzl

```diff
@@ -37,9 +37,9 @@ def proxy_wasm_cpp_host_dependencies():
         extra_target_triples = [
             "aarch64-unknown-linux-gnu",
             "wasm32-unknown-unknown",
-            "wasm32-wasi",
+            "wasm32-wasi",  # TODO: Change to wasm32-wasip1 once https://github.com/bazelbuild/rules_rust/issues/2782 is fixed
         ],
-        version = "1.68.0",
+        version = "1.79.0",
```

Comment:
It looks that `rules_rust` ships with Rust v1.77.2 and not v1.79.0, so you should use that version instead. Otherwise different targets are going to use different Rust versions.

Reply:
Oh wow didn't realize this wouldn't pin all usages to this version. Good call out!